src/utils/analytics.test.ts

```diff
@@ -0,0 +1,95 @@
+import { analyticsInstance, sendOffChainEvent, sendOnChainEvent } from '@utils/analytics'
+import AxiosMockAdapter from 'axios-mock-adapter'
+import { beforeEach, describe, expect, test, vi } from 'vitest'
+
+describe('analytics utils', () => {
+  const mockOnChainEvent = {
+    address: '0x123',
+    module: 'module',
+    action: 'action',
+    chainId: 1,
+    txHash: '0x123',
+  }
+  const mockOffChainEvent = {
+    action: 'action',
+    payload: { key: 'value' },
+  }
+
+  const mockInstance = new AxiosMockAdapter(analyticsInstance)
+
+  beforeEach(() => {
+    mockInstance.onPost('/events').reply(200, 'success bello')
+
+    vi.stubEnv('NODE_ENV', 'production')
```

Comment:
please add this to config: https://vitest.dev/config/#unstubenvs so that mocking env variable won't affect other test